railgun/manager/vars.go

```diff
@@ -0,0 +1,42 @@
+package manager
+
+// -----------------------------------------------------------------------------
+// Controller Manager - Constants & Vars
+// -----------------------------------------------------------------------------
+
+// HealthzPort is the default port the manager's health service listens on.
+// Changing this will result in a breaking change. Existing deployments may use the literal
+// port number in their liveness and readiness probes, and upgrading to a controller version
+// with a changed HealthzPort will result in crash loops until users update their probe config.
+// Note that there are several stock manifests in this repo that also use the literal port number. If you
+// update this value, search for the old port number and update the stock manifests also.
+const HealthzPort = 10254
+
+// MetricsPort is the default port the manager's metrics service listens on.
+// Similar to HealthzPort, it may be used in existing user deployment configurations, and its
+// literal value is used in several stock manifests, which must be updated along with this value.
+const MetricsPort = 10255
+
+// onOffUsage is used to indicate what textual options are used to enable or disable a feature.
+const onOffUsage = "Can be one of [enabled, disabled]."
```

Comment:
The usage of this const is strictly local to the definitions of `EnablementStatus` vars.
I suggest defining this const as a local one in `MakeFlagSetFor`, immediately above the block of `EnablementStatus` flags.

Reply:
sounds good, but it did need to be higher than that as other flags were using it. 1bc713e96791d9a0ed9b2a633a20e038e53fb5b0 moves it to the top of that file right before the flagsets.